src/services/sendPlio.js

```diff
@@ -9,6 +9,6 @@ export function sendPlio(isSingleEntryOnly, userID, redirectID) {
         unique_id: userID,
       });
       let fullurl = url + "?" + queryparams;
-      window.open(fullurl);
+      window.open(fullurl, "_self");
```

Comment:
`sendPlio`? Maybe this file should be called `redirectToPlio.js` 

Sorry... just keep noticing small things. You can ignore this :)

Reply:
I also think there is a better design for this.
In the folder `src/services/` there can be a file `redirection.js` that will handle all kinds of re-directions, be it plio, gmeet, zoom etc.

And in that file there can be one method called `redirectToPlio` along with other methods like `redirectToMeet` or `redirectToZoom` and those methods can then be imported into the vue component like you've done for `sendPlio`.

I just think a separate file just for redirection to plio is a little redundant?

Reply:
Yeah, I did do that from the next PR onwards. I have separation as there is more than one destination.

Reply:
You've done that in a PR? The advanced validation one? Or you have not committed that yet?

Reply:
I have committed the branch not made a PR though. The branch is `live-class`. Since there are more than one conditions, I abstracted the code for filtering redirection.